rtengine/dcraw.cc

```diff
@@ -923,7 +923,7 @@ ushort * CLASS ljpeg_row (int jrow, struct jhead *jh)
     }
     getbits(-1);
   }
-  FORC3 row[c] = (jh->row + ((jrow & 1) + 1) * (jh->wide*jh->clrs*((jrow+c) & 1)));
```

Comment:
This was changed by @heckflosse [here](https://github.com/Beep6581/RawTherapee/commit/f794cf684b18a9c2ca9b4ac350a10fd1ed37fc90#diff-3ca4dd32af70e9e6310586f5b5d92f55fd17859a3bc5bc9eeb4924c637846907L870). It looks like it was supposed to only be an optimization, so I guess it was just a lack of images using predictor modes > 1 back then that introduced the regression.